quickget

```diff
@@ -498,14 +498,17 @@ function releases_elementary() {
 }
 
 function releases_endeavouros() {
-    echo apollo_22_1 \
-    artemis-22_6 \
-    artemis_neo_22_7 \
-    artemis_neo_22_8 \
-    artemis_nova_22_9 \
-    atlantis-21_4 \
-    atlantis_neo-21_5 \
-    cassini_22_12
+    echo galileo-11-2023 \
+     cassini_nova-03-2023_r3 \
+     cassini_22_12 \
+     cassini_neo_22_12 \
+     artemis_nova_22_9 \
+     artemis_neo_22_8 \
+     artemis_neo_22_7 \
+     artemis-22_6 \
+     apollo_22_1 \
+     atlantis_neo-21_5 \
+     atlantis-21_4
```

Comment:
```suggestion
     local ENDEAVOUR_RELEASES="$(curl -s https://mirror.alpix.eu/endeavouros/iso/ | LC_ALL="en_US.UTF-8" sort -Mr | grep -o -P '(?<=<a href=").*(?=.iso">)' | grep -v 'x86_64' | cut -c 13-)"
    echo ${ENDEAVOUR_RELEASES,,}
```
This will sort by date.